### PR TITLE
Added .gitignore and .editorconfig files

### PR DESCRIPTION
**Pull request by pritamnegi**
#13

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+root = true
+
+# defaults
+[*]
+insert_final_newline = true     
+
+# 4 space indentation
+[*.{py}]
+indent_style = space
+indent_size = 4
```
Comment (satwikkansal) [inline .editorconfig]: Missing EOF

Comment (satwikkansal) [reply .editorconfig]: btw in future, you can check for this red sign which shows EOF is missing

**Comment by pritamnegi** [comment]
Please review the PR now @satwikkansal

**Comment by satwikkansal** [comment]
LGTM, merging it right away :tada:

**Pull request merged**